src/services/API/Event.js

```diff
@@ -2,6 +2,10 @@ import { apiClient } from "@/services/API/RootClient.js";
 import { eventsEndpoint } from "@/services/API/Endpoints.js";
 
 export default {
+  /**
+   *
```

Comment:
description for the method?

Reply:
As discussed, skippping it as method name is obvious